src/WebUI/Components/ResourceStatusComponent.tsx

```diff
@@ -1,32 +1,41 @@
-import * as React from "react";
-import { BaseComponent } from "@uifabric/utilities/lib";
-import { IStatusProps, Status, StatusSize } from "azure-devops-ui/Status";
-import { IVssComponentProperties } from "../Types";
-
-export interface IPodStatusProps extends IVssComponentProperties {
-    statusProps: IStatusProps | undefined;
-    statusSize?: StatusSize;
-    statusDescription?: string;
-    customDescription?: React.ReactNode;
-}
-
-export class PodStatusComponent extends BaseComponent<IPodStatusProps, {}> {
-
-    public render(): React.ReactNode {
-        return (
-            <div className="kube-status-container">
-                {
-                    this.props.statusProps &&
-                    <Status {...this.props.statusProps} size={this.props.statusSize || StatusSize.m} />
-                }
-                {
-                    this.props.statusDescription && 
-                    <span className="kube-status-desc">{this.props.statusDescription}</span>
-                }
-                {
-                    this.props.customDescription
-                }
-            </div>
-        );
-    }
+import * as React from "react";
+import { BaseComponent } from "@uifabric/utilities/lib";
+import { IStatusProps, Status, StatusSize } from "azure-devops-ui/Status";
+import { IVssComponentProperties } from "../Types";
+import { TooltipHost } from "azure-devops-ui/Tooltip";
+
+export interface IResourceStatusProps extends IVssComponentProperties {
+    statusProps: IStatusProps | undefined;
+    statusSize?: StatusSize;
+    statusDescription?: string;
+    customDescription?: React.ReactNode;
+    toolTipText?: string;
+}
+
+export class ResourceStatusComponent extends BaseComponent<IResourceStatusProps, {}> {
+
+    public render(): React.ReactNode {
+        return (
+            <div className="kube-status-container">
+                {
+                    this.props.toolTipText ?
+                        <TooltipHost content={this.props.toolTipText} >
```

Comment:
It will we good to use Tooltip from azure-devops-ui/TooltipEx, as TooltipHost has now been deprecated

Reply:
> It will we good to use Tooltip from azure-devops-ui/TooltipEx, as TooltipHost has now been deprecated

@dabasnidhi I'll file an issue and pick it up with another PR.